dbms/src/Storages/Transaction/SSTReader.cpp

```diff
@@ -62,7 +64,7 @@ MonoSSTReader::MonoSSTReader(const TiFlashRaftProxyHelper * proxy_helper_, SSTVi
     {
         auto && r = range->comparableKeys();
         auto start = r.first.key.toString();
-        LOG_DEBUG(log, "Seek cf {} to {}", static_cast<std::underlying_type<decltype(type)>::type>(type), Redact::keyToDebugString(start.data(), start.size()));
+        LOG_DEBUG(log, "Seek cf {} to {}", magic_enum::enum_name(type), Redact::keyToDebugString(start.data(), start.size()));
```

Comment:
also clean this
https://github.com/pingcap/tiflash/blob/b9899e9e302b4ad9b6aa4828f166fb9ec111530f/dbms/src/Storages/Transaction/SSTReader.h#L50-L54

Reply:
fixed